src/services/content-api/endpoints/class-endpoint-analytics-posts.php

```diff
@@ -93,29 +93,29 @@ public function get_endpoint(): string {
 	 * endpoint URL, if they are set. Since the Parse.ly API needs a key for
 	 * every value (e.g. tag=tag1&tag=tag2), we need to append them manually.
 	 *
-	 * @since 3.17.0
+	 * @param array<mixed> $query_args The arguments to pass to the API request.
 	 *
-	 * @param array<mixed> $args The arguments to pass to the API request.
 	 * @return string The endpoint URL for the API request.
+	 * @since 3.17.0
```

Comment:
Misplaces `@since`.

Reply:
Aghh damn PHPStorm always moving the `@since`tag  around. Fixing it.